### ISSUE TITLE
10.7.0 rc-1 .mkv 1080p files buggy playback

### ISSUE THREAD

**Issue by ASHuenchuleo** [jellyfin/jellyfin#4838]
**Describe the bug**
After updating to the rc-1, no .mkv files at a resolution of 1080p cant be played. The playback either stops completely or the browser reports that the file cannot be played. With some files it plays, then it displays the cover image for a few seconds, and it continues playing but the error appears anyway and playback is broken: subs don't load and syncplay is doesn't work.

**System (please complete the following information):**
 - OS: Hosted on Raspbian 32 bit for Raspberry Pi 4 8GB, also happens with Kubuntu 20
 - Virtualization: Docker
 - Clients: Browser, Android
 - Browser: Firefox 84, Chrome 87, Android app latest, Brave browser.
 - Jellyfin Version: 10.7.0 rc-1
 - Playback: Direct Play, Transcode
 - Installed Plugins: None
 - Reverse Proxy: nginx
 - Base URL: /jellyfin
 - Networking: Host
 - Storage: External USB drive

**To Reproduce**
1. Play this [test file](https://drive.google.com/file/d/0BwxFVkl63-lERkJRU003ZTd2VEk/view?usp=sharing) on jellyfin, the playback will fail without a message from the server. I couldn't find a testfile that can be played with a broken playback.

**Expected behavior**
The file should be played without any hiccups.

**Logs**
[Pastebin of the log when i tried to play the test file](https://pastebin.com/zydMaCb9). I'm not sure if it is good practice to use pastebin, if you need another format tell me.
**Screenshots**
[This is the information of one of the mkv files that don't play well. I sadly cannot share it and couln't find a test file where it happened.](https://user-images.githubusercontent.com/37780905/102738749-d0c19600-4329-11eb-8ee3-5656fcc2af35.png)

**Additional context**
This appears to be a server issue, since it happens in a lot of browsers and clients. I tried OpenMAX and VAPI transcoding on the raspberry pi server, and software encoding on my laptop, and the bug happens on both. I'm not quite sure what could be causing it but I hope this helps.

**Comment by nyanmisaka** [comment]
Seems to be an upstream issue?
https://github.com/video-dev/hls.js/issues/2893

**Comment by ASHuenchuleo** [comment]
Turns out the test video i uploaded doesn't play either in 10.6, so it appears to be a different issue.
I tested it a bit more and found out this is related to the reverse proxy (nginx). If I access jellyfin using server/jellyfin, the bug will occur with the original problematic files, but it doesn't happen when I access it using server:8096. Is there any new reverse proxy required configurations that I missed?
